draft-ietf-teas-actn-poi-applicability.txt

```diff
@@ -1014,6 +1014,30 @@ Internet-Draft                 ACTN POI                   November 2020
    [Editor's note:] Add YANG models used for tunnel and service
    configuration.
 
+3.3. PCEP
+
+   [RFC8623] examines the applicability of a Path Computation Element
+   (PCE) [RFC5440] and PCE Communication Protocol (PCEP) to the ACTN
+   framework. It further describes how the PCE architecture is
+   applicable to ACTN.  It also lists the PCEP extensions that are
+   needed to use PCEP as an ACTN interface.  The stateful PCE
+   [RFC8231], PCE-Initiation [RFC8281], stateful Hierarchical PCE (H-
+   PCE) [RFC8751], and PCE as a central controller (PCECC) [RFC8283]
+   are some of the key extensions that enable the use of PCE/PCEP for
+   ACTN.
+
+   Since the PCEP supports path computation in the packet as well as
+   optical networks, PCEP is well suited for inter-layer path
+   computation. [RFC5623] describes a framework for applying the PCE-
+   based architecture to interlayer (G)MPLS traffic engineering.
+   Further, the section 6.1 of [RFC8751] states the H-PCE applicability
+   for inter-layer or POI.
+
+   [RFC8623] lists various PCEP extensions that are applicable to ACTN.
+   It also list the PCEP extension for optical network and POI. Note
+   that the PCEP can be used in conjunction with the YANG models
+   described in the rest of this document.
+
```

Comment:
maybe we can spend few words to say how? it is true that you can you PCE for some things and YANG for others but what would be the benefit?

Reply:
**2020-11-09 TEAS**

ACTN can support multiple options at the MPI: it is possible to use only PCEP, as defined in RFC8637, or only the YANG models, as described in this document, or any possible combinations of PCEP and YANG models.

The scope of the text is just to clarify that all these possibilities exist.

For example, it is possible to use PCEP just for path computation and YANG models for the rest  (for example when PCEP has been already implemented/deployed).

@dancecca : are you ok to just add an example?

@sergiobelotti : please check if this example is reported in some public document